src/DiffEqUncertainty.jl

```diff
@@ -1,18 +1,38 @@
 module DiffEqUncertainty
 
-using DiffEqBase, Statistics, Distributions, Reexport
+# LinearAlgebra
+using DiffEqBase, Statistics, Reexport, RecursiveArrayTools, StaticArrays,
+    Distributions, KernelDensity, Zygote, LinearAlgebra, Random
+using Parameters: @unpack
+
 @reexport using Quadrature
-using KernelDensity
+import DiffEqBase: solve
+
+include("expectation/system_utils.jl")
+include("expectation/distribution_utils.jl")
+include("expectation/problem_types.jl")
+include("expectation/expectation.jl")
 
 include("probints.jl")
-include("koopman.jl")
+
 
 # Type Piracy, should upstream
 Base.eltype(K::UnivariateKDE)  = eltype(K.density)
 Base.minimum(K::UnivariateKDE) = minimum(K.x)
 Base.maximum(K::UnivariateKDE) = maximum(K.x)
+Base.extrema(K::UnivariateKDE) = minimum(K), maximum(K)
+
+Base.minimum(d::AbstractMvNormal) = fill(-Inf, length(d))
+Base.maximum(d::AbstractMvNormal) = fill(Inf, length(d))
+Base.extrema(d::AbstractMvNormal) = minimum(d), maximum(d)
+
+Base.minimum(d::Product) = minimum.(d.v)
```

Comment:
These are already defined in Distributions
https://github.com/JuliaStats/Distributions.jl/blob/master/src/multivariate/product.jl#L43

Reply:
I had these here as a stop gap until this PR when through https://github.com/JuliaStats/Distributions.jl/pull/1319